### ISSUE TITLE
Request for the pre-trained checkpoints

### ISSUE THREAD

**Issue by YuanfeiHuang** [jiexiaou/ShuffleFormer#1]
Interesting work! Can you release the pre-trained checkpoints for real image denoising?

**Comment by jiexiaou** [comment]
Thanks! We have uploaded the pretrained checkpoint for real image denoising.
